Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html

```diff
@@ -0,0 +1,36 @@
+<h2>Requirements</h2>
+<h3>To Run</h3>
+<p>
+    To run for either of the two positions of the Consulate, the player
+    must be in a nation within the Tanzanian Empire, and the nation must
+    be in the SMRF agreement, not hostile to Sudan, and approved by either
+    king of Sudan to run.
+</p>
+<h2>Power</h2>
+<h3>Voting</h3>
+<h4>Supreme Consul</h4>
+<p>
+    The Supreme Consul is to be supreme over the Grand Consul
+    and has the ability to veto anything the Grand Consul proposes 
+    up to five times between each election session. The Supreme Consul
+    may also do something called a Cogo te ad suffragium mecum ten times, or, 
+    "I force you to vote with me," shortened to Ctasm, which forces the
+    Grand Consul's vote (in to agree with the Supreme Consul's. The
+    Supreme Consul is to have one vote in the Ministry.
+</p>
+<h4>Grand Consul</h4>
+<p>
+    The Grand Consul is to have one vote in the Ministry and the ability to
+    transfer voting power in the ministry to the Supreme Consul for up to
+    one week at a time up to five times.
```

Comment:
If a term is one month and they can do this for up to a week at a time, wouldn't 5 times mean they can do it for, knger than their term? Or is ommiting per term done on purpose and the position can only ever do it 5 times? 
```suggestion
    one week at a time up to three times each term.
```

Reply:
yeah do that